src/components/molecules/DataResultItem.tsx

```diff
@@ -8,13 +8,24 @@ export type DataResultItemProps = {
     visualisationType: VisualisationType;
     title: string;
     description: string;
+    source: string;
+    published: string;
+    updated: string;
     tags: string[];
 };
 
-const DataResultItem: React.FC<DataResultItemProps> = ({ title, description, tags, visualisationType }) => (
+const DataResultItem: React.FC<DataResultItemProps> = ({
+    title,
+    description,
+    tags,
+    visualisationType,
+    source,
+    updated,
+    published,
```

Comment:
Might be easier to pass down the `metadata` object instead and specify what fields you require with 

```ts
type Props = Pick<MetadataEntry, 'title' | 'description' | ...>;
// or conversely Omit<MetadataEntry, '...' | '...' >;
```

That way you could create the DataResultItem as `<DataResultItem {...metadataEntry} />`

No need to change this now if you don't feel like it, but worth keeping in mind for later to avoid repetition.

Reply:
Good tip, I will keep that in mind! But I will keep it unchanged for now:):):)